vmdb/app/models/ems_kubernetes.rb

```diff
@@ -0,0 +1,21 @@
+class EmsKubernetes < EmsContainer
+  has_many :container_nodes,                      :foreign_key => :ems_id, :dependent => :destroy
+  has_many :container_groups,                       :foreign_key => :ems_id, :dependent => :destroy
+  has_many :container_services,                   :foreign_key => :ems_id, :dependent => :destroy
+
+  def self.ems_type
+    @ems_type ||= "kubernetes".freeze
+  end
+
+  def self.description
+    @description ||= "Kubernetes".freeze
+  end
+
+  def inventory
+    require 'kubeclient'
+    # TODO, timeout params
+    url = "http://#{hostname}:#{port}/api/"
+    client = Kubeclient::Client.new  url, "v1beta3"
+    client.get_all_entities
+  end
```

Comment:
I'd like to sit down and clean this up.  There is a certain pattern I'd like to follow where a `connect` method deals with connecting and returning the `Kubeclient::Client` instance, and then the caller (the parser) would call connect and it would call get_all_entries.  For now, this is how all the parsers work, so it keeps it consistent.

Also, we should not be building URIs by hand but instead using the URI class.  This helps with IPv6 support. cc @jrafanie